packages/ChangesReloaded-Core.package/CRBrowser.class/instance/hasSelectedChangeListItem.st

```diff
@@ -0,0 +1,4 @@
+accessing
+hasSelectedChangeListItem
+
+	^ self selectedChangeListItem isNil not
```

Comment:
`isNotNil`

Reply:
Surprisingly, `isNotNil` is not (hah!) a thing in Smalltalk/Squeak

Reply:
Oh wow, that's actually surprising. Could've sworn I read it somewhere…